common/client/node_lifecycle.go

```diff
@@ -134,6 +138,14 @@ func (n *node[CHAIN_ID, HEAD, RPC]) aliveLoop() {
 		lggr.Debug("Polling disabled")
 	}
 
+	var pollFinalizedHeadCh <-chan time.Time
+	if n.nodePoolCfg.FinalizedBlockPollInterval() > 0 {
+		lggr.Debugw("Finalized block polling enabled")
+		pollT := time.NewTicker(n.nodePoolCfg.FinalizedBlockPollInterval())
```

Comment:
Won't the NewTicker() panic if the parameter is 0?
I think in the fallback file, you should choose a default same as Eth mainnet, that's maybe 5 seconds?
Also, in config validation, ensure that this value is more than 0.

Reply:
> Won't the NewTicker() panic if the parameter is 0?

Yes, the NewTicker panics if the parameter is 0, that's why we do not initialize it unless provided value is > 0

> I think in the fallback file, you should choose a default same as Eth mainnet, that's maybe 5 seconds?

Sounds good.

> Also, in config validation, ensure that this value is more than 0.

IMHO, it should be possible to disable the check. Other health checks are optional, I do not see why this one should be an exception